pages/facts.html

```diff
@@ -4,9 +4,97 @@
   <meta charset="UTF-8">
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <title>Document</title>
+  <link rel="stylesheet" href="../style/style.css">
+  <link rel="stylesheet" href="../style/facts.css">
+  <title>Facts about plants </title>
+  <link rel="icon" href="../assets/plant-icon.png" type="image/png">
 </head>
 <body>
-  
+  <header>
+    <h1>Plant Party Fansite</h1>
+    <nav>
+      <ul>
+        <li>
+          <a class="menu-link" href="../index.html" target="_blank">Home</a>
+        </li>
+        <li>
+          <a class="menu-link" href="care.html" target="_blank">Care</a>
+        </li>
+        <li>
+          <a class="menu-link" href="#">Facts</a>
+        </li>
+        <li>
+          <a class="menu-link" href="gallery.html" target="_blank">Gallery</a>
+        </li>
+      </ul>
+    </nav>
+  </header>
+  <main>
+    <section>
```

Comment:
I could also see this page having 3 different sections, instead of just 1. 

One `section` for for lines 34-35, one `section` for the section titled "Facts", and another `section` for the section titled "Fun Facts"